### DISCUSSION TITLE
Handoff links from browser to native apps

### DISCUSSION THREAD

**Discussion by iGerman00** [stoatchat/discussions#530] [Feature Suggestions]
Right now to join a server you have to use the browser it seems, there is no link handoff like there is for Discord, I believe it's a really needed and small thing. For mobile apps, you can introduce URL schemes as well.

**Comment by insertish** [comment]
I believe this is already or at least partially implemented in the new Android app that's being worked on.

**Reply by iGerman00** [reply]
> I believe this is already or at least partially implemented in the new Android app that's being worked on.

Desktop definitely needs to have it as well, if you're going the invite URL path
